src/main/java/com/redhat/management/approval/EmailBody.java

```diff
@@ -162,15 +162,15 @@ private String getRequestContentLines(Map<String, Object> contents) {
             if (entry.getKey().equals(PARAMS_KEY))
                 continue;
   
-            String line = "<strong>" + customizeKey(entry.getKey()) + ":</strong>" + entry.getValue().toString() + "<br>";
+            String line = "<tr><th role="rowheader" scope="row">" + customizeKey(entry.getKey()) + ":</th></tr><td>" + entry.getValue().toString() + "</td></tr>";
```

Comment:
Compile error: need to escape these double quote: `"<tr><th role=\"rowheader\" scope=\"row\">"`